server/handlers/auth.ts

```diff
@@ -111,18 +111,58 @@ export const admin: Handler = async (req, res, next) => {
   throw new CustomError("Unauthorized", 401);
 };
 
+export const googleLogin: Handler = async (req, res, next) => {
+  const salt = await bcrypt.genSalt(12);
+  const password = await bcrypt.hash(req.body.password, salt);
+  const user = await query.user.find({ email: req.body.email });
+
+  if (user) {
+    if (!user?.verified) {
+      if (env.DISALLOW_VERIFICATION) {
+        await query.user.update({ email: req.body.email }, { verified: true })
+      } else {
+        await mail.verification(user);
+        return res
+          .status(201)
+          .send({ message: "Verification email has been sent." });
+      }
+    }
```

Comment:
Users created with Google should be verified instantly. So, if the user is not verified (for any unknown reason), it should verify it even if `DISALLOW_VERIFICATION == false`.
```suggestion
    if (!user?.verified) {
      await query.user.update({ email: req.body.email }, { verified: true })
    }
```